includes/wikia/VariablesBase.php

```diff
@@ -2637,6 +2637,40 @@
  */
 $wgFandomAppSmartBannerText = null;
 
+/**
+ * Used to map IntentX API wiki id to relevance key
+ */
+$wgFandomShopMap = [
+	'3510' => 'Assassin\'s Creed',
+	'532' => 'Avatar',
+	'613320' => 'Danganronpa',
+	'208733' => 'Dark Souls',
+	'599420' => 'Don\'t Starve',
+	'3035' => 'Fallout',
+	'1241752' => 'Fate/Grand Order',
+	'525179	' => 'Haikyuu!!',
+	'509' => 'Harry Potter',
+	'2233' => 'Marvel',
+	'113' => 'Star Trek',
+	'2125819' => 'Muppets',
+	'1242' => 'Nintendo',
+	'1081' => 'One Piece',
+	'74' => 'Pokemon',
+	'30404' => 'Red Dead',
+	'1249' => 'Resident Evil',
+	'4396' => 'Roblox',
+	'147' => 'Star Wars',
+	'621556' => 'Steven Universe',
+	'1006948' => 'Doctor Who',
+	'2569' => 'Thomas the Tank Engine',
+	'13346' => 'The Walking Dead',
+];
+
+$wgFandomShopMapDev = [
```

Comment:
You should use the dev.php config to set a different value of `$wgFandomShopMap` on dev instead of defining two variables.

Reply:
yep. thank you.